src/components/auth/formLogin2FA.js

```diff
@@ -0,0 +1,42 @@
+import { h } from 'preact';
+import FormSignU2F from './FormSignU2F';
+
+const formLogin2FA = ( { login2FA, twoFactorData: { isTOTP = true, U2F: U2FRequest } } ) => {
+  const model = {};
+  return (
+    <form onsubmit={e => (e.preventDefault(), login2FA(model))}>
+      {isTOTP && U2FRequest &&
+      <p> Choose one of the following 2FA methods </p>
```

Comment:
cad ?

Reply:
The two options are under (see picture)

![preview](https://user-images.githubusercontent.com/9265111/42036579-2f7484aa-7ae6-11e8-8d5e-346caea8b551.png)